src/sentry/search/events/datasets/metrics.py

```diff
@@ -276,3 +287,44 @@ def _resolve_percentile(
             ],
             alias,
         )
+
+    def _resolve_web_vital_function(
+        self,
+        args: Mapping[str, Union[str, Column, SelectType, int, float]],
+        alias: str,
+    ) -> SelectType:
+        column = args["column"]
+        metric_id = args["metric_id"]
+        quality = args["quality"].lower()
+
+        if column not in [
+            "measurements.lcp",
+            "measurements.fcp",
+            "measurements.fp",
+            "measurements.fid",
+            "measurements.cls",
+        ]:
+            raise InvalidSearchQuery("count_web_vitals only supports measurements")
+
+        measurement_rating = Column(f"tags[{indexer.resolve('measurement_rating')}]")
+
+        quality_id = indexer.resolve(quality)
+        if quality_id is None:
+            # Choose an invalid id so the result is 0
+            # TODO: Should be able to just not return a function at all in these cases
+            quality_id = 0
```

Comment:
Can you just return a stub that always resolves to 0 to save computation? like `toUint64(0)`?

Reply:
Good call, I was thinking to just not even create a function in these cases but that's a good solution that's far easier to implement